src/lib/update-alternatives/UI/alternatives_dialog.rb

```diff
@@ -47,25 +56,33 @@ def set_handler
       finish_dialog()
     end
 
+    def auto_handler
+      log.info("You have use the \"Set automatic mode\" button")
```

Comment:
it is good to log user selection, but it would be really nice to write to log also to which alternative it is selected. It really helps with later debugging.